File: onyx-database-tests/src/main/java/pojo/ListObject.java
package pojo;

import java.io.Serializable;
import java.util.List;

/**
 * Created by timothy.osborn on 4/14/15.
 */
public class ListObject implements Serializable
{
    public List objectArray;
    public List<Long> longArray;
    public List<Simple> simpleArray;

    public ListObject()
    {

    }
}

File: onyx-database-tests/src/main/kotlin/pojo/ListObject.kt
package pojo

import java.io.Serializable

/**
 * Created by timothy.osborn on 4/14/15.
 */
class ListObject : Serializable {
    var objectArray: MutableList<Any>? = null
    var longArray: MutableList<Long>? = null
    var simpleArray: MutableList<Simple>? = null
}
